examples/flax/language-modeling/requirements.txt

```diff
@@ -2,4 +2,4 @@ datasets >= 1.1.3
 jax>=0.2.8
 jaxlib>=0.1.59
 flax>=0.3.4
-optax>=0.0.8
+optax>=0.0.9
```

Comment:
adafactor is available in optax 0.0.9 https://github.com/deepmind/optax/releases